docs/Commands.md

```diff
@@ -169,17 +184,17 @@ FT.CREATE books-idx ON HASH PREFIX 1 book:details FILTER SCHEMA title TEXT categ
 
     * **SORTABLE**
 
-        Numeric, tag or text fields can have the optional SORTABLE argument that allows the user to later [sort the results by the value of this field](Sorting.md) (this adds memory overhead so do not declare it on large text fields).
+        Numeric, tag or text attributes can have the optional SORTABLE argument that allows the user to later [sort the results by the value of this field](Sorting.md) (this adds memory overhead so do not declare it on large text fields).
```

Comment:
GEO is also SORTABLE after [PR 2033](https://github.com/RediSearch/RediSearch/pull/2033)